libs/blocks/quiz/quiz.css

```diff
@@ -1,980 +1,384 @@
+.quiz-option.disabled .quiz-option-icon,
+.quiz-option.disabled .quiz-option-image,
+.quiz-option.disabled .quiz-option-text-container{
   opacity: 0.5;
 }
 
-.quiz-container .consonant-OneHalfCard-inner {
-  background-color: transparent;
-  border-radius: 10px;
-}
-
-.quiz-container .consonant-OneHalfCard-image {
-  margin-right: 16px;
-}
-
-.quiz-container .card.half-card.border.consonant-Card.consonant-OneHalfCard.quiz-option:hover {
-  background-color: var(--color-white);
-}
-
-.quiz-container .consonant-OneHalfCard-inner {
-  flex-direction: row;
-}  
-
-.quiz-container .consonant-OneHalfCard-cover {
-  width: 45px;
-  height: 45px;
-  object-fit: cover;
-  border-radius: 0.5rem;
-  margin-right: 1rem;
+.quiz-button-container {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  margin: 20px 0 24px;
 }
 
-.quiz-container .card.half-card.border.consonant-Card.consonant-OneHalfCard.quiz-option .icon {
-  width: 4rem;
-  height: 4rem;
+.quiz-button {
+  background: var(--color-white);
+  border: 2px solid var(--color-white);
+  border-radius: 30px;
+  cursor: pointer;
+  font-size: 14px;
+  font-weight: 700;
+  min-height: 32px;
+  min-width: 100%;
+  padding: 13px 86px;
 }
 
-.show {
-  display: block; 
+.quiz-button-label {
+  color: var(--link-hover-color-dark);
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 22px;
 }
 
-.hide {
-  display: none;
+.quiz-button[disabled] {
+  background: transparent;
+  cursor: not-allowed;
 }
 
-.quiz-page .footer-info {
-  margin-top: 12px;
+.quiz-button[disabled] .quiz-button-label{
+  color: var(--color-white);
 }
 
-/* Step Indicator CSS */
 .dot-indicators {
-  width: 300px;
-  top: 25px;
-  z-index: 100;
-  display: flex;
   align-items: center;
-  justify-content: space-between;
-}
-
-.dot-indicators:first-of-type {
-  display: none;
-}
-
-.dot-indicators:last-of-type {
   display: flex;
-  visibility: visible;
+  justify-content: space-between;
+  width: 300px;
+  z-index: 15;
 }
 
 .dot {
-  width: 100%;
   position: relative;
+  width: 100%;
 }
 
 .dot::before {
-  display: block;
-  content: "";
-  width: 13px;
-  height: 13px;
-  border-radius: 50%;
-  position: relative;  
-}
-
-.dot.current::before {
-  background-color: #000;
-}
-
-.dot.prev::before {
-  background-color: #868686;
-}
-
-.dot:not(.prev)::before {
   background-color: var(--color-white);
   border: 2px solid #676767;
-}
-
-.dot:last-of-type {
-  width:12px;
+  border-radius: 50%;
+  content: '';
+  display: block;
+  height: 13px;
+  width: 13px;
 }
 
 .dot::after {
-  display: block;
-  content: ' ';
-  width: calc(100% - 23px);
-  height: 2px;
   border-bottom: 2px dashed #676767;
+  content: '';
+  height: 2px;
+  margin-left: 20px;
   position: absolute;
   top: 6px;
-  margin-left: 20px;
-}
-
-/* Visited lines */
-.dot.prev:not(.current)::after {
-  border-bottom: 2px solid #676767;
+  width: calc(100% - 23px);
```

Comment:
Again, it might make sense to add a comment or switch to a relevant variable name for the `23px` value

Reply:
added  a comment on this, it's basically the total width of .dot minus the width of the next indicator dot to prevent the dashed lines from spilling out over the next dot.